src/App/Containers/Todos/TableTodo.js

```diff
@@ -2,28 +2,28 @@ import React from 'react'
 
 import { Table } from 'reactstrap'
 
-const TodoTableModule = ({ todoDatas }) => {
-  const renderRows = todoDatas.map((singleDatas) => (
+const TodoTableModule = ({ slicedData }) => {
+  const renderRows = slicedData.map((singleDatas) => (
```

Comment:
Add more meaningful naming conventions, like const renderRows = slicedData.map((todo) => (
<li> {todo.title} </li>
<li> {todo.desc} </li>
)

We know the data we are getting is todo. so we can use todo as name for single entity.